tests/callbacks/test_gpu_stats_monitor.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
let's add a test that user gets a friendly error when he tries to tun in on machine without GPU

Reply:
Already added :)